src/openvic-simulation/economy/production/ProductionType.hpp

```diff
@@ -83,6 +83,7 @@ namespace OpenVic {
 	private:
 		IdentifierRegistry<ProductionType> IDENTIFIER_REGISTRY(production_type);
 		PopType::sprite_t PROPERTY(rgo_owner_sprite);
+		IndexedMap<GoodDefinition, ProductionType const*> PROPERTY(good_to_rgo_production_type);
```

Comment:
IMO it's better to store this as a variable in each `GoodDefinition` rather than a map here, that way we can simply get the production type from the good rather than need access to the `ProductionTypeManager`

Reply:
That doesn't work as the identifier registry only exposes consts.
I'd have to extend the registry macro to expose mutable methods.

If that is the desired approach, it should be done as a follow up PR.